test/Validation.test.js

```diff
@@ -255,6 +255,74 @@ describe("Validation", () => {
 				" - configuration.stats should be one of these:"
 			);
 		}
+	}, {
+		name: "Invalid plugin provided: bool",
+		config: {
+			entry: "foo.js",
+			plugins: [
+				false
+			]
+		},
+		test(err) {
```

Comment:
Could you use `message` instead of `test` to test the complete message?

Like all other tests here except one.

Reply:
yeah sure, nice catch :)